modules/installation-vsphere-machines.adoc

```diff
@@ -141,20 +141,20 @@ Ensure that all virtual machine names across a vSphere installation are unique.
 
 .. On the *Select a compute resource* tab, select the name of a host in your datacenter.
 +
-.. On the *Select clone options*, select
-*Customize this virtual machine's hardware*.
+.. On the *Select clone options*, select *Customize this virtual machine's hardware*.
```

Comment:
`On the *Select clone options*`

Not sure if `the` should be removed or a noun (`tab`?) should be added here

Reply:
I went with "menu item" as I'm not sure if they are tabs. To me, "Advanced Parameters" is a tab.